packages/cli/src/cmds/validator/keymanager/decryptKeystoreDefinitions/types.ts

```diff
@@ -0,0 +1,14 @@
+import {Logger} from "@lodestar/utils";
+import {LocalKeystoreDefinition} from "../interface.js";
+
+export type DecryptKeystoreWorkerAPI = {
+  decryptKeystoreDefinition({keystorePath, password}: LocalKeystoreDefinition, force: boolean): Promise<Uint8Array>;
```

Comment:
why is the `force` param needed? seems to be unused in worker

Reply:
the force param is used to override the LOCK file protection, should be used by the worker

Reply:
Its no longer used by the worker, its handled in the main thread. It can be removed. And we can rename the option there.